common-rest/src/main/java/org/uniprot/api/common/repository/stream/store/uniprotkb/UniProtKBBatchStoreIterableUtil.java

```diff
@@ -0,0 +1,42 @@
+package org.uniprot.api.common.repository.stream.store.uniprotkb;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import org.uniprot.core.taxonomy.TaxonomyLineage;
+import org.uniprot.core.uniprotkb.UniProtKBEntry;
+import org.uniprot.core.uniprotkb.impl.UniProtKBEntryBuilder;
+import org.uniprot.core.uniprotkb.taxonomy.Organism;
+import org.uniprot.core.util.Utils;
+
+public class UniProtKBBatchStoreIterableUtil {
+
+    public static List<UniProtKBEntry> populateLineageInEntry(
```

Comment:
test missing?

Reply:
we have it at UniProtKBBatchStoreIterableUtilTest.java